scale/docs/algorithm_integration/step4.rst

```diff
@@ -1,234 +1,81 @@
-                "required": true, 
-                "type": "file", 
-                "name": "output_file_tif2"
-            },
-            {
-                "media_type": "text/csv", 
-                "required": true, 
-                "type": "file", 
-                "name": "output_file_csv"
-            }
-        ], 
-        "shared_resources": [], 
-        "command_arguments": "${input_file} ${job_output_dir}", 
-        "version": "1.0", 
-        "command": "/app/my_wrapper.sh"
-    },
-
-Lines 27-36:  The "input_data" section defines the inputs of the job that will be managed by Scale.  The "media_type"
-defines the expected file_type of the input while the "type" keyword is a string defining whether in input is a file,
-list of files, or a property.  The "required" field is optional and indicates if the input is an optional command
-argument.  *If an input is optional, the job will still execute except replace the input with an empty string.* The
-"name" field is a user-defined unique string.  **These input names match the variables in the command_arguments**
-                
-Lines 37-56:  The "output_data" section defines the expected outputs of the job.  The "media_type" defines the expected
-file type of output while the "type" keyword defines whether it expects a file or a list of files.  The "required"
-keyword is a boolean used if the output is guaranteed to be produced by the algorithm on a successful run.  By default,
-the "required" keyword is set to true.  The "name" keyword must be a unique string defining the output.
-**These names are matched with the names in the results manifest to capture results**
-                
-Line 57:      The "shared_resources" keyword is currently unused and reserved for future usage.  It should be set to [].
-
-Line 58:      The "command_arguments" keyword defines what inputs are passed into the docker container.  You can think
-of the arguments as appended to the end of the contents of your dockerfile's ENTRYPOINT for what command is executed.
-The variables here **MUST** match the "name"(s) in your "input_data" section.  The special variable "${job_output_dir}"
-is supplied by Scale and will be your empty output directory Scale will use to search for your results_manifest.json.
-
-Line 59:      The "version" is an optional string defining the version of the definition specification used.  This
-defaults to the latest version.
-
-Line 60:      The "command" field defines the main command to execute on the command line.
-*This should match your ENTRYPOINT in your dockerfile*.
+Adding your new job into a Scale job type
+=========================================
+
+After you have successfully tested your Docker container and inserted the job into the database, you can add the job to an existing recipe or create a brand new recipe. Recipes allow you to chain the inputs/outputs of one job to another job to create a workflow for your algorithm processing.
```

Comment:
Reword to talk about creating a job type and then adding the job type to a new/existing recipe?

Reply:
🤔 Perhaps something like:
>After you have successfully tested your Seed job and published it to your Docker registry, you are ready to add a new job type to Scale. Once the job type is defined, you will be able to start running it by adding it to an existing recipe, or by creating a brand new recipe.